src/components/ui4/BlurBackground.tsx

```diff
@@ -11,5 +13,14 @@ export interface BlurBackgroundProps {
 export const BlurBackground = (props: BlurBackgroundProps) => {
   const { overlayColor = 'rgba(0, 0, 0, 0)' } = props
   const theme = useTheme()
-  return <BlurView blurType={theme.isDark ? 'dark' : 'light'} style={StyleSheet.absoluteFill} overlayColor={overlayColor} />
+  const styles = getStyles(theme)
+
+  return <BlurView blurType={theme.isDark ? 'dark' : 'light'} style={styles.blurView} overlayColor={overlayColor} />
 }
+
+const getStyles = cacheStyles(() => ({
+  blurView: {
+    ...StyleSheet.absoluteFillObject,
+    backgroundColor: isAndroid ? '#00000055' : undefined
+  }
+}))
```

Comment:
This is all wrong. By default, the `BlurView` looks consistent on both iOS and Android. However, Android lets you manually adjust the tint color using the `overlayColor`, so we ourselves started making Android look different. But now we don't like the inconsistency, so we tweak it back a different way!!!

What about just deleting the `overlayColor` prop and letting things automatically be the same on both platforms? This will also deal with the light-mode, which your current solution doesn't address.

Reply:
overlayColor applied on the whole screen on Android. That was the reason for the transparent value.

Reply:
Oh, interesting. So it's a bug-fix to the blur library!? In that case, we should document it as such.